### ISSUE TITLE
Package Updates: Displaying changelog

### ISSUE THREAD

**Issue by qooob** [webmin/webmin#480]
Jamie, hi!

When new updates arrived sometimes you want to know what are the changes. Most packages provide _changelog_ on the inside. It would be absolutely _**cool**_ if we could also see the change log, when clicking on the package.

It should be done in the `textarea` tag with `readonly` attribute and limited height.

Here is the [example](https://fedora.pkgs.org/25/fedora-updates-x86_64/bind-9.10.4-3.P5.fc25.x86_64.rpm.html). Click changelog tab.

**Comment by jcameron** [comment]
Webmin actually does this already, on systems using YUM if the `yum-changelog` package is installed.
Is there a command to get the changelog on debian though?

**Comment by qooob** [comment]
I don't know about Debian really. I actually meant latest RHEL systems that use `dnf` command.

`dnf updateinfo info` is the command that must be used with latest RHEL systems, where `yum` is replaced by `dnf`. It will display the text of update advisory notices for all updates at once - very convenient. This saying, I wish we could add into the default opener (that is used across Webmin all over), right at the bottom of the table of updates (In case there is info). In case opener is done the same way as before in previous modules (like Actions Log), the theme will automatically make it look great.

Could you try?

![screenshot from 2017-01-15 11-56-09](https://cloud.githubusercontent.com/assets/4426533/21961427/908aacaa-db1a-11e6-96fe-77bb993ec920.png)

**Comment by jcameron** [comment]
Which specific Linux distro and version is this on?

**Comment by qooob** [comment]
Fedora 25 Server edition

**Comment by jcameron** [comment]
Ok, try out the patch I just attached to this ticket.

**Comment by swelljoe** [comment]
Minor FYI: RHEL and CentOS are still on yum, even in version 7. I imagine RHEL/CentOS 8 will use dnf.

**Comment by jcameron** [comment]
Yeah, this patch should use whichever of YUM or DNF is installed (like other functions in Webmin already do).
